src/storage.ts

```diff
@@ -14,12 +14,16 @@
  *  limitations under the License.
  */
 
+import * as firebase from 'firebase/storage';
 import { openDB, DBSchema, IDBPDatabase } from 'idb';
 import { v4 as uuidv4 } from 'uuid';
-import { ref, uploadBytes, UploadMetadata, UploadResult } from 'firebase/storage';
+
 import { firebaseStorage } from './firebase';
 
 const kMainDBName = 'appDB';
+const firebaseDir = 'public';
+const remoteIdPrefix = 'R-';
```

Comment:
style nit: we should be consistent w/ constant naming, so either all k-prefix, or none. (kFirebaseDir, kRemoteIdPrefix)

Reply:
Done.